tox.ini

```diff
@@ -87,11 +87,11 @@ description = Run high availability integration tests
 pass_env =
     {[testenv]pass_env}
     CI
-    CI_PACKED_CHARMS
 deps =
     pytest
     juju==3.2.0.1
     pytest-mock
+    git+https://github.com/canonical/data-platform-workflows@v5.1.2\#subdirectory=python/pytest_plugins/pytest_operator_cache
```

Comment:
is this comment (if it is a comment) required?

Reply:
No, it is not a comment which is why we have `\` :) 

I copied this from [these instructions](https://github.com/canonical/data-platform-workflows/blob/v4.0.0/.github/workflows/build_charms_with_cache.md#step-2-install-plugin-for-pytest-operator) with the help of @carlcsaposs-canonical